deploy/osd-managed-resources/managed-namespaces.ConfigMap.yaml

```diff
@@ -19,6 +19,8 @@ data:
       - name: openshift-codeready-workspaces
       - name: openshift-custom-domains-operator
       - name: openshift-customer-monitoring
+      - name: openshift-logging
```

Comment:
This is a creep in scope and not something included in OSD-12049.  This should be removed and addressed in a separate ticket if desired.

Reply:
Manually removed for now.
OSD-12049 has been created to discuss that.